exercises/knapsack/description.md

```diff
@@ -0,0 +1,28 @@
+In this exercise, lets try to solve a classic problem.
+
+Bob is a thief. After months of planning, he 
+finally manages to crack the security systems of a 
+high-class apartment. In front of him are many 
+items, each with a value (v) and weight (w). Bob of
+course wants to maximise the total value he can 
```

Comment:
```suggestion
course wants to maximize the total value he can 
```

Reply:
Is there are British/American language policy on Exercism?

Reply:
Not that I have bloody heard, mate!  But I think we allow them.

Reply:
I didn't realize `maximise` was British.  Most words are are easier to spot IE. harbor / harbour.  OK how do I force Chrome to use American and British dictionaries?.....  OK done.  Now Chrome knows the Queen's English.  😃   Carry on.